s30130-EmilGulaj/pamietnik.txt

```diff
@@ -0,0 +1,68 @@
+2024-03-10
+Moj kochany pamietniczku:
+
+Opis zajec
+	To byly zajecia co prawda wprowadzajace do samego przedmiotu, 
+	jak i do pierwszego zadania ktore było omawiane na wykladzie,
+	ale i tak były ciekawe.
+
+Czego sie nauczyles
+	Dowiedzialem sie jak bardzo duzo nie wiem, i jak to czego nie wiem 
+	jest mi potrzebne.
+
+Co jest dla Ciebie jeszcze nie jasne i musisz poglebic temat
+	Zdecydowanie musze nauczyc sie tego jak dziala Git/Github,
+	bo nie mam o tym zielonego pojecia, a wydaje sie byc bardzo
```

Comment:
To powinno być już opanowane na zajęciach z przedmiotu UKOS w poprzednim semestrze.